new_sem3/dz3_ten_words.py

```diff
@@ -1,3 +1,69 @@
+    'New in version 3.7: Descriptors for nested definitions. They are accessed through the new children attribute. Each has a new parent attribute.\n'
+    'The descriptors returned by these functions are instances of Function and Class classes. Users are not expected to create instances of these classes.\n'
+    'Function Objects\n'
+    'Class Function instances describe functions defined by def statements. They have the following attributes:\n'
+    'Function.file\n'
+    'Name of the file in which the function is defined.\n'
+    'Function.module\n'
+    'The name of the module defining the function described.\n'
+    'Function.name\n'
+    'The name of the function.\n'
+    'Function.lineno\n'
+    'The line number in the file where the definition starts.\n'
+    'Function.parent\n'
+    'For top-level functions, None. For nested functions, the parent.\n'
+    'New in version 3.7.\n'
+    'Function.children\n'
+    'A dictionary mapping names to descriptors for nested functions and classes.\n'
+    'New in version 3.7.\n'
+    'Function.is_async\n'
+    'True for functions that are defined with the async prefix, False otherwise.\n'
+    'New in version 3.10.\n'
+    'Class Objects\n'
+    'Class Class instances describe classes defined by class statements. They have the same attributes as Functions and two more.\n'
+    'Class.file\n'
+    'Name of the file in which the class is defined.\n'
+    'Class.module\n'
+    'The name of the module defining the class described.\n'
+    'Class.name\n'
+    'The name of the class.\n'
+    'Class.lineno\n'
+    'The line number in the file where the definition starts.\n'
+    'Class.parent\n'
+    'For top-level classes, None. For nested classes, the parent.\n'
+    'New in version 3.7.\n'
+    'Class.children\n'
+    'A dictionary mapping names to descriptors for nested functions and classes.\n'
+    'New in version 3.7.\n'
+    'Class.super\n'
+    'A list of Class objects which describe the immediate base classes of the class being described. Classes which are named as superclasses but which are not discoverable by readmodule_ex() are listed as a string with the class name instead of as Class objects.\n'
+    'Class.methods\n'
+    'A dictionary mapping method names to line numbers. This can be derived from the newer children dictionary, but remains for back-compatibility.')
+
+s2 = re.sub('[\d]+','', re.sub('[^\w]+',' ', s)) # Не учитываем знаки препинания и цифры.
+s2 = s2.lower().split(' ') # Не учитываем регистр и разбиваем на список.
+s2 = list(filter(lambda x: x != '', s2)) # Фильтруем список, убирая пустые значения
```

Comment:
Считается, что так быстрее
```suggestion
s2 = [*filter(lambda x: x != '', s2)] # Фильтруем список, убирая пустые значения
```